app/models/player.rb

```diff
@@ -1,5 +1,6 @@
 class Player < ApplicationRecord
   belongs_to :tournament
-  has_many :matchups
+  has_many :matchups, dependent: :destroy
```

Comment:
Not sure we want to destroy matchups when we destroy a player, since we need to keep a log of all matchups to accumulate wins/losses and data

Reply:
We'll add an active boolean to the players table